README.md

```diff
@@ -49,15 +49,19 @@ notify-pass credentials/mmg
 
 ### Postgres
 
-Install [Postgres.app](http://postgresapp.com/).
+Postgres is built in the CONCOURSE_TESTS as an extra layer derived from the TEST layer.
+This is built to allow the tests to run using the API PRODUCTION layer.
 
-The API (on Notify) works with PostgreSQL 15. After installation, open the Postgres app, open the sidebar, and update or replace the default server with a compatible version.
-
-**Note:** you may need to add the following directory to your PATH in order to bootstrap the app.
+#### How to build and run tests locally:
 
 ```
-export PATH=${PATH}:/Applications/Postgres.app/Contents/Versions/15/bin/
+1. run the make command
+    `make run-local-tests-with-docker`
 ```
+The step above will build the image, run the container, start postgresql and run the tests.
+
+Note: There is a file called "version.py.dist" that is used for "healthcheck.py". As it stands, this file is unusable and the local build will fail. You must, either rename it to "version.py" or simply create a local file with that name and copy the contents to it.
```

Comment:
This shouldn't really need to be done manually. We have a `make generate-version-file` step that creates this file. This step also automatically runs when `make bootstrap` is run, which is what devs and Concourse use to prepare the app to run or be tested.

I'm also not sure that `version.py.dist` _is_ actually used anywhere. I think it's just a leftover file maybe - can't see any references to it at least...

Reply:
Agree that we should remove the need for manual action like this please.

Reply:
ok